app/screens/interactive_dialog/dialog_element.tsx

```diff
@@ -22,6 +22,21 @@ function selectKeyboardType(type: InteractiveDialogElementType, subtype?: Intera
     return selectKB(subtype);
 }
 
+function getStringValue(value: string | number | boolean | string[] | undefined): string | undefined {
```

Comment:
Any value (heh, pun intended) in converting `false` to string and gets "false"?

```
    if (typeof value === 'number' || typeof value === 'boolean') {
        return value.toString();
    }
```

Reply:
Maybe if I understand correctly, the initValues function would have converted that.

Reply:
```suggestion
function getStringValue(value?: string | number | boolean | string[]): string | undefined {
```

Reply:
True/false values should only be used on Boolean settings, and those will use the `getBooleanValue`. If a boolean value gets into this point, probably is because something is "wrong", and `undefined` is the safe approach.

But I don't have a strong opinion on this. Nevertheless, it "should never happen".


Regarding the change to `getStringValue`, I would argue the argument is not optional. You must provide an argument, just that that argument can be undefined. 0/5 on that too, so if you have a strong opinion, happy to change it.

Reply:
I was thinking of consistency.  Since if people can pass undefined, it's almost like not 
 passing anything.  But it is silly to see `getStringValue()`.  I rather see people send `getStringValue(undefined)` instead of `getStringValue()`.

Long to short: don't change. :P